src/testenv/testAsArDefaultResolver.py

```diff
@@ -28,11 +28,13 @@
 """
 
 import os
+import shutil
+import unittest
+
 from pxr import Ar
+
 from rdo import ReplaceResolver
 
```

Comment:
Please keep `unittest` and `shutil` modules! This file must stay as much as possible the same of the original `testArDefaultResolver.py` (basically only resolver names are changed). This is because we want to check that our resolver always behave like a defaultResolver.

Reply:
I think those have been reordered by `black`, but they are still part of the module

Reply:
Ah I see! I think we should not run black on this file then.